### PR TITLE
changed wording

### PR DESCRIPTION
**Pull request by ghost**
makes it sound better. some of the wording is not the best.

### CODE REVIEW

**Comment by Jamesharris10** [comment]
#### SUMMARY
Just changes wording to sound nicer

##### ISSUE TYPE
- Docs Pull Request

File: README.rst [hunk 1]
```diff
@@ -7,13 +7,13 @@ Ansible
...
 ad-hoc task execution, network automation, and multi-node orchestration. Ansible makes complex
-changes like zero-downtime rolling updates with load balancers easy. More information on `the Ansible website <https://ansible.com/>`_.
+changes like zero-downtime rolling updates with load balancers easy. More information on the ansible `website <https://ansible.com/>`_.
 
 Design Principles
 =================
 
-*  Have a dead-simple setup process with a minimal learning curve.
-*  Manage machines very quickly and in parallel.
+*  Have a extremly simple setup process with a minimal learning curve.
```
Comment (acozine) [inline README.rst:15]:
```suggestion
*  Have an extremely simple setup process with a minimal learning curve.
```

File: README.rst [hunk 2]
```diff
@@ -7,13 +7,13 @@ Ansible
 Ansible is a radically simple IT automation system. It handles
 configuration management, application deployment, cloud provisioning,
 ad-hoc task execution, network automation, and multi-node orchestration. Ansible makes complex
-changes like zero-downtime rolling updates with load balancers easy. More information on `the Ansible website <https://ansible.com/>`_.
+changes like zero-downtime rolling updates with load balancers easy. More information on the ansible `website <https://ansible.com/>`_.
```
Comment (acozine) [inline README.rst:10]: We do like to capitalize the name, unless we're referring directly to the repo or the CLI command. 
```suggestion
changes like zero-downtime rolling updates with load balancers easy. More information on the Ansible `website <https://ansible.com/>`_.
```

File: README.rst [hunk 3]
```diff
@@ -7,13 +7,13 @@ Ansible
...
 
 Design Principles
 =================
 
-*  Have a dead-simple setup process with a minimal learning curve.
-*  Manage machines very quickly and in parallel.
+*  Have a extremly simple setup process with a minimal learning curve.
+*  Manage machines fast and in parallel.
```
Comment (acozine) [inline README.rst:16]: Fast is an adjective, quickly is an adverb, we need an adverb here.
```suggestion
*  Manage machines quickly and in parallel.
```

**Review by acozine — COMMENTED** [review]
Thanks @Jamesharris10 for contributing to the Ansible documentation! You found some things we should update, indeed.

I've made some suggestions on your PR - if you accept them, I can merge this change.

Thanks again!

**Comment by Jamesharris10** [comment]
Fine with me.

**Comment by acozine** [comment]
@Jamesharris10 great! It's your PR, so you need to accept the changes - see https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request for instructions. Once you've committed the suggested changes, our CI system will run again. Once CI passes, we can merge this PR.

**Comment by acozine** [comment]
Closing in favor of #75115. I'd give credit to the person previously known as @Jamesharris10 if I could!

**Pull request closed without merging**